doc/ug/getting_started_dv.md

```diff
@@ -88,3 +88,27 @@ Please refer to the [checklist]({{< relref "doc/project/checklist" >}}) to under
 
 The [UART DV](https://github.com/lowRISC/opentitan/tree/master/hw/ip/uart/dv) area can be used as a canonical example for making progress.
 If it is not clear on how to proceed, feel free to file an issue requesting assistance.
+
+## Reproduce a DV failure in CI
+
+Follow these steps to reproduce the failure
+
+1. Make sure the version of VCS is the same as the [one](https://github.com/lowRISC/opentitan-private-ci/blob/master/jobs.yml#L5) running in CI.
+
+2. CI runs against an auto-generated merge commit, which effectively is generated by merging the pull request (PR) into the master branch.
```

Comment:
sorry this is not related to this PR exactly, but i'm sort of confused by this.
Does this mean what CI runs is not the same as what I have locally when I push the PR?

Because when it says "merged" into master, that almost makes me think it gets rebased on to tip of tree or something.

Reply:
right, very likely what CI runs is not the same as you local has. it gets rebased on to tip of tree and I think that makes more sense. Your local branch can be very old. I didn't know it until Philipp told me.